Sources/App/Settings/Security/SecurityViewController.swift

```diff
@@ -0,0 +1,28 @@
+import Eureka
+import FirebaseInstallations
+import FirebaseMessaging
+import PromiseKit
+import RealmSwift
+import Shared
+import UIKit
+
+class SecurityViewController: HAFormViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = L10n.SettingsDetails.General.Security.title
+
+        form
+            +++ Section()
+            <<< SwitchRow("switch", {
+                $0.title = L10n.SettingsDetails.General.Security.action
+                $0.value = Current.settingsStore.biometricsRequired
+                $0.onChange { row in
+                    Current.settingsStore.biometricsRequired = row.value ?? false
```

Comment:
since settings may be open when locked, can you make it require authentication to disable?

Reply:
I remove catalyst implementation and kept it just for iOS for now to keep it simple and iterate afterwards. 
So in iOS the user has to unlock the app before going to settings.